content/en/tracing/setup_overview/open_standards/_index.md

```diff
@@ -93,6 +93,371 @@ service:
       exporters: [datadog/api]
 ```
 
+### Environment Specific Setup
+
+#### Host:
+
+- Download the appropriate binary from [the project repository latest release](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/latest).
+
+- Create a `otel_collector_config.yaml` file. Here is an example template to get started. It enables the collector's otlp receiver and datadog exporter.
+
+- Run on the host with the configration yaml file set via the `--config` parameter. For example,
+
+  ```
+    curl -L https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/latest/download/otelcontribcol_linux_amd64 | otelcontribcol_linux_amd64 --config otel_collector_config.yaml
+  ```
+
+#### Docker
```

Comment:
I think we can mostly get rid of this section and just link here instead: https://opentelemetry.io/docs/collector/getting-started/#docker, making clear that we are present in the **contrib** flavor.

I don't think we should teach people how to use Docker or how to use the OpenTelemetry Collector in Datadog docs, we should keep this for Datadog specific configuration

Reply:
These sections are quite misleading (note the `kill $pid1; docker stop otelcol
` at the end of the docker command, for example) and have the user cloning the repo locally. i think it's better to have working examples in our docs and then contrib upstream to improve the upstream docs, and when those are improved we can point to them directly.

Reply:
Makes sense, let's keep it then. Can you open an issue for this on the https://github.com/open-telemetry/opentelemetry.io repo?

Reply:
added https://github.com/open-telemetry/opentelemetry.io/issues/370

Reply:
also added for rbac: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/2274